modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -56,20 +61,27 @@ spec:
     resources:
       requests:
         storage: <fs_volume_size> <2>
-  osImages: <3>
-    - openshiftVersion: "<ocp_version>" <4>
-      version: "<ocp_release_version>" <5>
-      url: "<iso_url>" <6>
-      rootFSUrl: "<root_fs_url>" <7>
+  imageStorage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: <is_volume_size> <3>      
+  osImages: <4>
+    - openshiftVersion: "<ocp_version>" <5>
+      version: "<ocp_release_version>" <6>
+      url: "<iso_url>" <7>
+      rootFSUrl: "<root_fs_url>" <8>
       cpuArchitecture: "x86_64"
 ----
 <1> Volume size for the `databaseStorage` field, for example `10Gi`.
 <2> Volume size for the `filesystemStorage` field, for example `20Gi`.
-<3> List of OS image details. Example describes a single {product-title} OS version.
-<4> {product-title} version to install, for example, `4.8`.
-<5> Specific install version, for example, `47.83.202103251640-0`.
-<6> ISO url, for example, `https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.7/rhcos-4.7.7-x86_64-live.x86_64.iso`.
-<7> Root FS image URL, for example, `https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.7/rhcos-live-rootfs.x86_64.img`
+<3> Volume size for the `imageStorage` field, for example `2Gi`.
+<4> List of OS image details. Example describes a single {product-title} OS version.
```

Comment:
And I know that this isn't in the scope of your change, but "Example describes a..." could be rephrased if you so choose. :)